### PR TITLE
Rename BoringTun to NepTUN

### PR DESCRIPTION
**Pull request by jjanowsk**

### CODE REVIEW

File: neptun-cli/Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 [package]
-name = "boringtun-cli"
+name = "neptun-cli"
 description = "an implementation of the WireGuard® protocol designed for portability and speed"
 version = "0.6.0"
 authors = ["Noah Kennedy <nkennedy@cloudflare.com>", "Andy Grover <agrover@cloudflare.com>", "Jeff Hiner <jhiner@cloudflare.com>"]
 license = "BSD-3-Clause"
-repository = "https://github.com/cloudflare/boringtun"
-documentation = "https://docs.rs/boringtun/0.5.2/boringtun/"
```
Comment (gytsto) [inline neptun-cli/Cargo.toml:8]: no more docs?

Comment (jjanowsk) [reply neptun-cli/Cargo.toml:8]: Not until we start publishing to crates.io

File: neptun/src/device/api.rs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2024 Nord Security. All rights reserved.
 // Copyright (c) 2019 Cloudflare, Inc. All rights reserved.
```
[the remark this answers is not in the document]
Comment (jjanowsk) [reply neptun/src/device/api.rs]: Updated

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Build, test and lint locally
+The local feedback cycle is faster than waiting for the CI. Make sure your changes can be built locally and that tests, rustfmt and clippy all pass locally. A green CI is a happy CI.
+
+### PR Hygiene
+On top of the CI being green, every PR will go through code review, and you can help us speed up the review process by making your PR easier to review. Here are some guidelines:
+
+**Small PRs are easier to review than big PRs**, so try to keep your PRs small and focused. To achieve that, try to make sure you PR doesn't contain multiple unrelated changes and if you are doing some bigger feature work, try to split the work into multiple smaller PRs that solve the problem together.
+
+**A clean history can make things easier**. Some PRs are easier to review commit-by-commit, rather than looking at the full changelist in one go. To enable that, prefer `rebase` over `merge` when updating your branch. Keeping PRs small and short-lived will also help keep your history clean since there's less time for upstream to change that much.
```
Comment (Hasan6979) [inline CONTRIBUTING.md:20]: Regarding changelog are we going for the git-cliff ?

File: neptun-cli/src/main.rs [hunk 4]
```diff
@@ -136,18 +137,18 @@ fn main() {
             })) => {
                 let mut b = [0u8; 1];
                 if sock2.recv(&mut b).is_ok() && b[0] == 1 {
-                    println!("BoringTun started successfully");
+                    println!("NepTUN started successfully");
                     exit(first_child_exit_code)
                 } else {
-                    eprintln!("BoringTun failed to start");
+                    eprintln!("NepTUN failed to start");
```
Comment (Hasan6979) [inline neptun-cli/src/main.rs:143]: nit: Are we going for `NepTun` similar to `BoringTun` or `NepTUN` ?

**Review by Hasan6979 — COMMENTED** [review]
Other than a few questions. +1

File: neptun-cli/src/main.rs (hunk 4 above)
Comment (jjanowsk) [reply neptun-cli/src/main.rs:143]: NepTUN

File: CONTRIBUTING.md (hunk 3 above)
Comment (jjanowsk) [reply CONTRIBUTING.md:20]: I don't think there was a decision made. I think it was left to be investigated. Once we decide to go with it we should note it here

**Review by gytsto — APPROVED** [review]

**Pull request merged**